js/block.js

```diff
@@ -4297,6 +4297,7 @@ function Block(protoblock, blocks, overrideName) {
                         80,
                         84,
                         88,
+                        90,
```

Comment:
no need to add 90 here...

Reply:
You can set selectedValue as the block value (this.value). But you need to make sure it is >=40 and <=208 and make sure it is divisible by 4.

something like
selectedValue = Math.floor(this.value / 4) * 4;
if (selectedValue < 40) selectedValue = 40;
if (selectedValue > 208) selectedValue = 208;